libpod/oci.go

```diff
@@ -414,6 +414,9 @@ func (r *OCIRuntime) stopContainer(ctr *Container, timeout int64) error {
 		stopSignal = uint(syscall.SIGTERM)
 	}
 
+	if timeout == 0 {
```

Comment:
I really don't like this. 0 timeout already has a meaning in the code - stop immediately with SIGKILL

Reply:
Fixed